exocom-server/features/support/world.ls

```diff
@@ -116,7 +116,26 @@ World = !->
       expected.timestamp = that if received-message.timestamp
       expected.response-time = that if received-message.response-time
       expected.response-to = that if response-to
-      jsdiff-console received-message, expected, done
+      expect(received-message).to.include expected
+      expect(received-message.session-id).to.match /^\w|-*$/
+      done!
+
+
+  @verify-equal-session-id = ({client-name, message-name, id = '123', response-to}, done) ~>
+    service = @service-mocks[client-name]
+    wait-until (~> service.received-messages[0]?.name && service.received-messages.length == 2), ~>
```

Comment:
This (`service.received-messages.length == 2`) is tied a little too closely to some of the other step definitions. Let's do `service.received-messages.length > 1`. Also, why are we checking for `.name` here?